app/assets/stylesheets/components/data-portal/c-slider.scss

```diff
@@ -0,0 +1,50 @@
+
+.c-slider {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  transform: translate(calc(100% + 1px), 0);
```

Comment:
For some reason, when I moved the slide to show/hide it, there was always 1px of the slider showing up. This fixes it.